### ISSUE TITLE
Invisible Underscore in Microsoft Office 2010

### ISSUE THREAD

**Issue by DejavuDmitry** [FreeRDP/FreeRDP#2634]
I found strange bug:

``` bash
$ cat /etc/os-release 
NAME="Ubuntu"
VERSION="14.10 (Utopic Unicorn)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 14.10"
VERSION_ID="14.10"
HOME_URL="http://www.ubuntu.com/"
SUPPORT_URL="http://help.ubuntu.com/"
BUG_REPORT_URL="http://bugs.launchpad.net/ubuntu/"
$ uname -a
Linux to-l-chuprina 3.16.0-36-generic #48-Ubuntu SMP Tue Apr 14 20:07:19 UTC 2015 i686 i686 i686 GNU/Linux
$ ~/build/FreeRDP/client/X11/xfreerdp --version
This is FreeRDP version 1.2.4-dev (git 1810656)
```

![2015-05-20-140034_1920x1080_scrot](https://cloud.githubusercontent.com/assets/9697445/7724863/fadcd9fc-fefd-11e4-9953-dfee1b177c5d.png)
![2015-05-20-140030_1920x1080_scrot](https://cloud.githubusercontent.com/assets/9697445/7724864/023a478e-fefe-11e4-8a32-5fc9e7936762.png)

**Comment by hardening** [comment]
@DejavuDmitry is it something that appeared after 181065619932b60b5d6895842d15f47d304a027d ? Or were you experimenting the same before this commit ?

Note: IIRC we have asked some questions in other issues, without any answers so far ...

**Comment by nfedera** [comment]
@DejavuDmitry please let us know ...
1. Answers to questions from @hardening 
2. What is the remote OS you are you connecting to ?
3. What is your complete xfreerdp command line ?
4. Does adding /gdi:sw "fix" the issue ?

**Comment by nfedera** [comment]
I've just tested using Word via Office 2010 Starter on Windows 7 SP1 x64 using Times New Roman (size 16) and could not reproduce the issue (tested with and before 1810656).
@DejavuDmitry so we really need all the requested details please. And maybe you could upload an example word document.

**Comment by DejavuDmitry** [comment]
@nfedera 
1. I don't know
2. 
```bash
>systeminfo

Имя узла:                       XXXXXXXXXXX
Название ОС:                    MicrosoftR Windows ServerR 2008 Standard
Версия ОС:                      6.0.6002 Service Pack 2 сборка 6002
Изготовитель ОС:                Microsoft Corporation
Параметры ОС:                   Рядовой сервер
Сборка ОС:                      Multiprocessor Free
Зарегистрированный владелец:    Пользователь Windows
Зарегистрированная организация:
Код продукта:                   XXXXXXXXXXXXXXXXX
Дата установки:                 14.04.2015, 13:37:51
Время загрузки системы:         13.05.2015, 12:06:34
Изготовитель системы:           Bochs
Модель системы:                 Bochs
Тип системы:                    x64-based PC
Процессор(ы):                   Число процессоров - 2.
                                [01]: Intel64 Family 15 Model 6 Stepping 1 Genui
neIntel ~2900 МГц
                                [02]: Intel64 Family 15 Model 6 Stepping 1 Genui
neIntel ~2901 МГц
Версия BIOS:                    Bochs Bochs, 01.01.2011
Папка Windows:                  C:\Windows
Системная папка:                C:\Windows\system32
Устройство загрузки:            \Device\HarddiskVolume1
Язык системы:                   ru;Русский
Язык ввода:                     ru;Русский
Часовой пояс:                   (GMT+02:00) Вильнюс, Киев, Рига, София, Таллин,
Хельсинки
Полный объем физической памяти: 16 383 МБ
Доступная физическая память:    13 069 МБ
Файл подкачки: Макс. размер:    32 570 МБ
Файл подкачки: Доступен:        29 320 МБ
Файл подкачки: Используется:    3 250 МБ
Расположение файла подкачки:    C:\pagefile.sys
Домен:                          tgo.local
Сервер входа в сеть:            \\XXXXXXXX
Исправление(я):                 Число установленных исправлений - 1.
                                [01]: KB955430
```
My cmnd line

```bash
~/build/FreeRDP/client/X11/xfreerdp /t:"$LABEL-$REMOTE_USER" +clipboard /cert-ignore /size:${X_RES}x${Y_RES} /drive:FLASH,/media/"$USER" /drive:HOME,"$HOME" /u:"$REMOTE_USER" /p:"$REMOTE_PASS" /v:"$REMOTE_HOST" /printer:
```

With  /gdi:sw

```bash
/home/dejavu/bin/rdp: строка 34: 20169 segmentation fault                   (подготовлен дамп ядра) ~/build/FreeRDP/client/X11/xfreerdp /gdi:sw /t:"$LABEL-$REMOTE_USER" +clipboard /cert-ignore /size:${X_RES}x${Y_RES} /drive:FLASH,/media/"$USER" /drive:HOME,"$HOME" /u:"$REMOTE_USER" /p:"$REMOTE_PASS" /v:"$REMOTE_HOST" /printer:
```

**Comment by akallabeth** [comment]
Color conversion and GDI fixes in current master 3a9a2b944c428cde13e26d6012f51eee9aed0541 should have fixed any of these issues.
If you encounter them again feel free to reopen this issue.
